web-console/src/utils/sampler.mock.ts

```diff
@@ -29,9 +38,45 @@ This data is the returned sample when ingested with:
 {"timestamp":"2016-04-11T09:22:00Z","user":"Alice","followers":3,"spend":5.1,"id":"73534533","tags":["a","b"],"nums":[7,8]}
  */
 
-export const JSON_SAMPLE: SampleHeaderAndRows = {
-  header: ['timestamp', 'user', 'followers', 'spend', 'id', 'tags', 'nums'],
-  rows: [
+export const JSON_SAMPLE: SampleResponse = {
+  numRowsRead: 3,
+  numRowsIndexed: 3,
+  logicalDimensions: [
+    { type: 'string', name: 'user', multiValueHandling: 'SORTED_ARRAY', createBitmapIndex: true },
+    {
+      type: 'long',
+      name: 'followers',
+      multiValueHandling: 'SORTED_ARRAY',
+      createBitmapIndex: false,
+    },
+    { type: 'json', name: 'spend', multiValueHandling: 'SORTED_ARRAY', createBitmapIndex: true },
+    { type: 'string', name: 'id', multiValueHandling: 'SORTED_ARRAY', createBitmapIndex: true },
+    { type: 'json', name: 'tags', multiValueHandling: 'SORTED_ARRAY', createBitmapIndex: true },
+    { type: 'json', name: 'nums', multiValueHandling: 'SORTED_ARRAY', createBitmapIndex: true },
+  ],
+  physicalDimensions: [
+    { type: 'json', name: 'user', multiValueHandling: 'SORTED_ARRAY', createBitmapIndex: true },
+    {
+      type: 'json',
+      name: 'followers',
+      multiValueHandling: 'SORTED_ARRAY',
+      createBitmapIndex: true,
+    },
+    { type: 'json', name: 'spend', multiValueHandling: 'SORTED_ARRAY', createBitmapIndex: true },
+    { type: 'json', name: 'id', multiValueHandling: 'SORTED_ARRAY', createBitmapIndex: true },
+    { type: 'json', name: 'tags', multiValueHandling: 'SORTED_ARRAY', createBitmapIndex: true },
+    { type: 'json', name: 'nums', multiValueHandling: 'SORTED_ARRAY', createBitmapIndex: true },
```

Comment:
i suppose I changed this underneath you in #14014, but it might be nice to update this at some point (also I have a bug to fix here since right now the sampler will show the new 'auto' in the physical schema but still have 'json' in the logical schema, so i think its ok to hold off on updating this)